### ISSUE TITLE
Nav element should not have role="search"

### ISSUE THREAD

**Issue by will0684** [gcdevops/HRWhiteListing#186]
This fails WCAG best practice of using correct aria roles on appropriate elements. You can see the element and selector here:

<img width="669" alt="Screen Shot 2020-04-14 at 12 47 03 PM" src="https://user-images.githubusercontent.com/31868510/79251843-b9ebab00-7e4e-11ea-8505-2921d15c973e.png">

You can read more here:

https://dequeuniversity.com/rules/axe/3.5/aria-allowed-role
